index.html

```diff
@@ -1183,6 +1183,110 @@
 preferences or other heuristics in providing or disabling functionality could also contribute to a
 browser fingerprint.
 
+## Information access {#information}
+
+The many APIs available to websites expose lots of data that can be combined into
+information about people, web servers, and other things. We can divide that
+information into three categories:
+
+1. Information that's fine to expose, for example because a person or group with
+    sufficient authority intended to expose that information or to do something that
+    necessarily exposes the information, or because it's not about people at all. For
+    example:
+
+    * The geolocation and camera APIs ask whether a person wants to expose their data.
+    * The URL a person is visiting must be sent to a server in order to navigate to that
+       URL, and known private-information-retrieval methods are too expensive to avoid that
+       exposure.
```

Comment:
What does "known private-information-retrieval methods" mean here?

It seems that at least some user agents are interested in potentially modifying the URL before sending it onward and some of that would not be expensive. (It remains to be seen whether it will be effective.)

Reply:
I'm thinking of https://en.wikipedia.org/wiki/Private_information_retrieval, which makes it theoretically possible to retrieve a URL without telling the server which URL you're getting, but IIUC requires the server to compute over their entire space of resources in order to return one.

Modifying the URL doesn't seem to solve the problem of the server learning which resource the user wants. I guess you could modify it into pointing at a different resource, but users seem unlikely to be happy with that. 🤪